ci-operator/config/openshift-assisted/assisted-installer-deployment/OWNERS

```diff
@@ -34,7 +34,6 @@ filters:
     - ori-amizur
     - osherdp
     - romfreiman
-    - slaviered
```

Comment:
it will get overridden from the repos, so I think there's no point to do it here